### DISCUSSION TITLE
Preview not showing

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#16291] [Q&A]
Hi,
I enabled [OrchardCore.ContentPreview](https://docs.orchardcore.net/en/main/reference/modules/ContentPreview/) module and then I added PreviewPart to a content type. When I click on "Preview button", it always shows a blank page.
Which step have I missed?
 
<img width="1120" alt="Screenshot 2024-06-11 at 13 54 48" src="https://github.com/OrchardCMS/OrchardCore/assets/61869142/e542f1ea-7004-401e-abdc-01588eb06289">

**Comment by Piedone** [comment]
Is this with a full CMS app? You don't need to use Preview Part then.

**Reply by DotCat1985** [reply]
It's a full CMS app.

**Reply by Piedone** [reply]
Then I suggest removing the Preview Part :).

**Comment by hishamco** [comment]
@DotCat1985 please navigate the page the render the content type that you click the preview button for it
